synfig-studio/src/gui/app.h

```diff
@@ -130,6 +130,8 @@ class App : public Gtk::Main, private IconController
 	*/
 
 public:
+	static Glib::RefPtr<App> create();
+	void init(const synfig::String& basepath, int *argc, char ***argv);
```

Comment:
Should "init" be public?

Reply:
This part of the code is also suggested by ice0 ! Kindly confirm it once if I have to change this ! Thanks.

Reply:
> Should "init" be public?
@rodolforg 
Yes, because it called from the lambda in `main()` and we cannot hide it.

@DhairyaBahl 
How do you propose to change it?

Reply:
@ice0 Oops ! My bad ! You are right, it have to be public coz we are accessing it in the main function ! Although I changed constructor and destructor to protected from public ! but according to me we can't change visibility of init